tests/test_validate_call.py

```diff
@@ -113,17 +121,9 @@ def __call__(self, x: int) -> int:
 
 
 def test_invalid_signature() -> None:
-    # In some versions, these functions may not have a valid signature
-    for func in (max, min, breakpoint, sorted, compile, print, [].append, {}.popitem, int().bit_length):
-        try:
-            inspect.signature(func)
-            assert validate_call(func).__name__ == func.__name__
-            assert validate_call(func).__qualname__ == func.__qualname__
-            assert validate_call(partial(func)).__name__ == f'partial({func.__name__})'
-            assert validate_call(partial(func)).__qualname__ == f'partial({func.__qualname__})'
-        except ValueError:
-            with pytest.raises(PydanticUserError, match=(f"Input function `{func}` doesn't have a valid signature")):
-                validate_call(func)
+    # Builtins functions not supported:
+    with pytest.raises(PydanticUserError, match=(f'Input built-in function `{breakpoint}` is not supported')):
+        validate_call(breakpoint)
```

Comment:
There's no real value in parametrizing as they are all the same type (a builtin function). This would be equivalent to something like:

```python
@pytest.mark.parametrize('value', [1, 2, 3])
def test(value):
    assert isinstance(value, int)
```